.github/workflows/analysis.yml

```diff
@@ -41,10 +41,13 @@ jobs:
           ./phpcs --version
 
       - name: Run PHP_CodeSniffer
-        run: ./phpcs
+        run: |
+          ./phpcs --runtime-set ignore_warnings_on_exit true
 
       - name: Run PHP CS Fixer
+        if: always()
```

Comment:
Why not use:
```
continue-on-error: true
```
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

Reply:
Yes... it looks better. I'll fix this conditional, thanks :)

Reply:
But... job status always successful. Conditionals in steps doesn't change step status.
Main idea - is run all checks and see all reports from PHP tools... then developer can fix those problem with one fix commit.
So... I'll revert to explicit conditions :)

Reply:
Ah, wasn’t expecting that status will change